climaf/derived_variables/__init__.py

```diff
@@ -0,0 +1 @@
+from ipsl_derived_variables import *
```

Comment:
no need to "import *"
should be written 
  import ipsl_derived_variables

Reply:
I've made the correction according to your comment and tested it.
We can however discuss whether those derived variables definitions should remain center-specific, or if we could define some of them as general definitions.

Reply:
Right. When porting CESEMP for CNRM, I have prepared a version where only IGCMOUT-dependent definitions are under the control of flag atIPSL. I will use it and maybe change te file name